webpack.config.js

```diff
@@ -9,6 +9,7 @@ var base = {
         host: '0.0.0.0',
         port: process.env.PORT || 8073
     },
+    devtool: 'source-map',
```

Comment:
Works for me. And when I said "I'm always able to figure out where the error is occurring" I didn't mean because it gives the source mapping, just because if I click to the line in the inspector, it's clear which file I'm looking at from the context since we don't minify in development.